packages/flutter/lib/src/material/grid_tile_bar.dart

```diff
@@ -67,11 +67,8 @@ class GridTileBar extends StatelessWidget {
       end: trailing != null ? 8.0 : 16.0,
     );
 
-    final ThemeData theme = Theme.of(context);
     final ThemeData darkTheme = ThemeData(
```

Comment:
Should this just be ThemeData.dark()?

Reply:
As you pointed out in chat, they are the same. I just updated to fix this.